lab5.py

```diff
@@ -0,0 +1,59 @@
+"""Modules are absent"""
+
+
+class TimeConverter:
+    """
+    Main function
+    """
+    def __init__(self, _time: str, hours=None, minutes=None, seconds=None):
+        try:
+            _hours, _minutes, _seconds = _time.split(":")
+            self.hours = int(_hours)
+            self.minutes = int(_minutes)
+            self.seconds = int(_seconds)
+        except ValueError:
+            self.hours = hours
+            self.minutes = minutes
+            self.seconds = seconds
```

Comment:
looks like a magic code here- person with limited experience will spend to much time to figure out the idea behind of this try-except. Such logic requires to be covered in docsting